x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -143,20 +142,24 @@ export const AllCasesList = React.memo<AllCasesListProps>(
 
     const onFilterChangedCallback = useCallback(
       (newFilterOptions: Partial<FilterOptions>) => {
-        if (newFilterOptions.status && newFilterOptions.status === CaseStatuses.closed) {
+        if (
+          newFilterOptions.status &&
```

Comment:
nit: I think we can remove one line with optional chaining: `if (newFilterOptions?.status === CaseStatuses.closed && ...)` because undefined won't equal closed.